src/util/LocalStorageWorker.ts

```diff
@@ -16,18 +16,36 @@
 
 import { EventMessage } from '../models/EventMessage';
 
+const persistedPinnedMessageKeyPart = 'pinnedMessage';
 class LocalStorageWorker {
 	getPersistedPinnedMessages(): EventMessage[] {
 		try {
-			const stringPersistedPinnedMessages = localStorage.getItem('pinnedMessages');
-			return stringPersistedPinnedMessages ? JSON.parse(stringPersistedPinnedMessages) : [];
+			const persistedPinnedMessages: EventMessage[] = [];
```

Comment:
i don't like idea of keeping every pinned message separately, why don't we just store them inside of collection?